parent/pom.xml

```diff
@@ -28,7 +28,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
         <kafka.version>2.8.0</kafka.version>
-        <camel.version>3.19.0-SNAPSHOT</camel.version>
+        <camel.version>3.18.1</camel.version>
```

Comment:
This change is out of the scope of this PR, but it seems that CI build keeps failing with
```
Error:  Failed to execute goal on project camel-aws-cloudwatch-sink-kafka-connector: Could not resolve dependencies for project org.apache.camel.kafkaconnector:camel-aws-cloudwatch-sink-kafka-connector:jar:3.19.0-SNAPSHOT: The following artifacts could not be resolved: org.apache.camel:camel-aws2-cw:jar:3.19.0-SNAPSHOT, org.apache.camel:camel-core:jar:3.19.0-SNAPSHOT, org.apache.camel:camel-yaml-dsl:jar:3.19.0-SNAPSHOT: Could not find artifact org.apache.camel:camel-aws2-cw:jar:3.19.0-SNAPSHOT in apache.snapshots (https://repository.apache.org/snapshots/) -> [Help 1]
```
The version `3.19.0` doesn't work due to missing pom property `azure-jackson2-version`.
I also find it bit confusing to have version `3.19.0-SNAPSHOT` with parent [camel-dependencies](https://github.com/apache/camel-kafka-connector/blob/main/pom.xml#L26) v3.18.1.

Anyway I'm not that familiar with whole maven setup, I just wanted to check if it help pass the build.